ChangeLog.md

```diff
@@ -2,6 +2,36 @@
 
 ## Unreleased
 
+- The client now uses the V2 GRPC API exposed by the node. This introduces some breaking
+  changes:
+    - Since the node serves the V2 GRPC API at port 20000 by default, the default value of
+      the `--grpc-port` option has been updated to reflect this.
```

Comment:
We should remove --grpc-token and mention it here.

Reply:
I assume by this you refer to `--grpc-authentication-token`?